packages/amplify_core/lib/amplify_core.dart

```diff
@@ -15,9 +15,31 @@
 
 library amplify_core;
 
+/// Common types
+export 'package:aws_common/aws_common.dart';
```

Comment:
Yes, it does apply here. I think this is a case where we want to obscure the existence of aws_common so that the end-user only needs to import amplify_flutter to get the functionality of amplify_core/aws_common as well.